Content/posts/how-the-swift-package-index-project-got-started.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2021-05-05 12:00
+title: How the Swift Package Index project got started because of a button
+description: How did the Swift Package Index project get started, and why does a button feature so prominently in the story? Read on to find out.
+---
+
+It’s been [one year since we made the first commit](https://twitter.com/_sa_s/status/1386033811348197380) on the [Swift Package Index repository](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server), and we think that deserves a little celebration!
+
+<center style="font-size: 30px;">🎂</center>
+
+Did you enjoy the cake? 😂 No? Well, how about we also tell you the story of how this project started?
+
+**Note:** We usually write blog posts here in the third person as this project is very much a joint effort, but this is a story with lots of references to both [Dave](https://twitter.com/daveverwer) and [Sven](https://twitter.com/_sa_s) individually, so I’ll write it in the first person. _Dave_
```

Comment:
I think you meant `first person plural` as our usual perspective?

Reply:
Fixed.